src/gcodePlanner.cpp

```diff
@@ -263,7 +263,7 @@ GCodePath& GCodePlanner::addTravel(Point p)
         combed = comb->calc(lastPosition, p, combPaths, was_inside, is_inside, retraction_config.retraction_min_travel_distance, via_outside_makes_combing_fail, fail_on_unavoidable_obstacles);
         if (combed)
         {
-            bool retract = combPaths.size() > 1;
+            bool retract = always_retract || combPaths.size() > 1;
```

Comment:
In the description of this PR and ithis parameter you refer to it as always retracting, while it only retracts when combing. This is rather confusing.

I think you should always apply the retraction.
You can set `path = getLatestPathWithConfig(&travel_config, SpaceFillType::None);` at the beginning of this function and always use that `path` as well to call `addTravel_simple(p, path)` while already having set `path.retract = true` for that path.